exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -380,93 +388,81 @@ func (b *BTCMarkets) ModifyOrder(action *order.Modify) (string, error) {
 }
 
 // CancelOrder cancels an order by its corresponding ID number
-func (b *BTCMarkets) CancelOrder(order *order.Cancel) error {
-	orderIDInt, err := strconv.ParseInt(order.OrderID, 10, 64)
-	if err != nil {
-		return err
-	}
-
-	_, err = b.CancelExistingOrder([]int64{orderIDInt})
+func (b *BTCMarkets) CancelOrder(o *order.Cancel) error {
+	_, err := b.RemoveOrder(o.OrderID)
 	return err
 }
 
 // CancelAllOrders cancels all orders associated with a currency pair
 func (b *BTCMarkets) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, error) {
-	cancelAllOrdersResponse := order.CancelAllResponse{
-		Status: make(map[string]string),
-	}
-	openOrders, err := b.GetOpenOrders()
+	var resp order.CancelAllResponse
+	tempMap := make(map[string]string)
+	orders, err := b.GetOrders("")
 	if err != nil {
-		return cancelAllOrdersResponse, err
-	}
-
-	var orderList []int64
-	for i := range openOrders {
-		orderList = append(orderList, openOrders[i].ID)
+		return resp, err
 	}
-
-	if len(orderList) > 0 {
-		orders, err := b.CancelExistingOrder(orderList)
+	for x := range orders {
```

Comment:
You could replace this with 
https://api.btcmarkets.net/doc/v3#tag/Batch-Order-APIs/paths/~1v3~1batchorders~1{ids}/delete

This will minimise RESTful calls and greatly speed things up